ansible/plugins/filter/filters.py

```diff
@@ -43,11 +43,6 @@ def match_key(value, dictionary, raise_error=True, feedback_name='os'):
     return False
 
 
-def latest_version(versions):
-    versions.sort(key=lambda s: map(int, s.split('.')))
```

Comment:
THIS was the Python3 problem that's been haunting me, stopping me from doing any of this Docker update stuff on my Mac, having to do it all on my Linux machine instead. I've lost the error, although it's been mentioned a couple of times in this repo already, something about `<` not being compatible with `map`.
So I've avoided the Python2/3 dumpster fire entirely and used good 'ol reliable Bash to get the job done, see below.